dashboard/static/js/controllers.js

```diff
@@ -26,18 +28,59 @@ visApp.controller('mainController', ['$scope', 'Get','$location', '$anchorScroll
             $anchorScroll();
         });
     }
+    // get the user default selection from DB and if not available selects the first item from the dropdown list
+    get_default_selection = function () {
+        let get_defaults_url = '/api/v1/courses/' + dashboard.course_id + '/get_user_default_selection?default_type=assignment';
```

Comment:
is "let" necessary here?

Reply:
`let` has a local scope so it makes sense to use that here within the function scope